Common/cpp/headers/SharedItems/ShareableValue.h

```diff
@@ -13,6 +13,19 @@ using namespace facebook;
 
 namespace reanimated {
 
+struct HostFunctionHandler {
```

Comment:
why even add this? It adds more complexity and uses more memory just to make an access to `functionName` easier, but that name's only used once so I don't really see a point in doing this.

Reply:
discussed internally, resolving